src/lib/decodeData.ts

```diff
@@ -109,16 +111,25 @@ export const isValidTuple = (valueType: string, valueContent: string) => {
       );
     }
 
-    if (
-      valueContentParts[i].slice(0, 2) === '0x' &&
-      !isHex(valueContentParts[i])
-    ) {
-      throw new Error(
-        `Invalid tuple for valueType: ${valueType} / valueContent: ${valueContent}. valueContent of type: ${valueContentParts[i]} is not a valid hex value`,
-      );
-    }
+    const isValueContentHexLiteral = valueContentParts[i].slice(0, 2) === '0x';
```

Comment:
Maybe you can use `.startsWith` instead.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith